x-pack/plugins/security_solution/public/common/components/navigation/helpers.ts

```diff
@@ -20,28 +20,31 @@ import {
 } from '../url_state/helpers';
 
 import { SearchNavTab } from './types';
-import { SourcererUrlState } from '../../store/sourcerer/model';
 import { useIsExperimentalFeatureEnabled } from '../../hooks/use_experimental_features';
 import { useUiSetting$ } from '../../lib/kibana';
 import { ENABLE_GROUPED_NAVIGATION } from '../../../../common/constants';
 
-export const getSearch = (tab: SearchNavTab, urlState: UrlState): string => {
+export const getSearch = (
+  tab: SearchNavTab,
+  urlState: UrlState,
+  globalQueryString: string
+): string => {
   if (tab && tab.urlKey != null && !isAdministration(tab.urlKey)) {
-    return getUrlStateSearch(urlState);
+    // TODO: Temporary code while we are migrating all query strings to global_query_string_manager
```

Comment:
Is there an issue to track this follow up work?

Reply:
I created this one:  https://github.com/elastic/kibana/issues/135012